.github/workflows/build-dev-image.yml

```
@@ -0,0 +1,127 @@
+    inputs:
+      branch:
+        description: 'Branch to build from'
+        required: true
+        default: 'main'
+        type: string
+
+jobs:
+  build-dev-image:
+    name: Build Dev Docker Image
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)